chaincode/compute_plan_dag.go

```diff
@@ -75,7 +75,7 @@ func (dag *ComputeDAG) sort() error {
 			current[i].Depth = depth
 			final = append(final, current[i])
 			if _, ok := IDPresents[current[i].ID]; ok {
-				return fmt.Errorf("compute plan error: Duplicate training task ID: %s", current[i].ID)
+				return errors.Internal("compute plan error: Duplicate training task ID: %s", current[i].ID)
```

Comment:
This error seems to be a conflict, no?

Reply:
Yes it's a conflict problem... Will update it

Reply:
Done